opencti-platform/opencti-graphql/src/modules/vocabulary/vocabulary-utils.ts

```diff
@@ -110,16 +110,16 @@ export const openVocabularies: Record<VocabularyCategory, Array<{ key: string, d
   ],
   // C
   case_severity_ov: [
-    { key: '01 - low', description: 'Low impact', aliases: ['low'] },
-    { key: '02 - medium', description: 'Medium impact', aliases: ['medium'] },
-    { key: '03 - high', description: 'High impact', aliases: ['high'] },
-    { key: '04 - critical', description: 'Critical impact', aliases: ['critical'] },
+    { key: 'low', description: 'Low impact', aliases: ['low'], order: 1 },
+    { key: 'medium', description: 'Medium impact', aliases: ['medium'], order: 2 },
+    { key: 'high', description: 'High impact', aliases: ['high'], order: 3 },
+    { key: 'critical', description: 'Critical impact', aliases: ['critical'], order: 4 },
```

Comment:
why did the key change ? since we have default orders here too, don't we want them to be updated in the database ?